### PR TITLE
Display more details including street address and custom fields

### PR DESCRIPTION
**Pull request by niklasva82**
Display more details including street address and custom fields if available.

### CODE REVIEW

File: src/components/panes/TargetPane.jsx [hunk 1]
```diff
@@ -21,23 +22,39 @@ export default class TargetPane extends PaneBase {
...
+                ).toArray();
+
+            info = info.concat(customFields);
         }
 
         let map = null;
         if (target.get('zip_code') && target.get('city')) {
             let qs = querystring.stringify({
-                center: target.get('zip_code') + '+' + target.get('city'),
+                center: target.get('street_address') + '+' + target.get('zip_code') + '+' + target.get('city'),
```
Comment (richardolsson) [inline src/components/panes/TargetPane.jsx:57]: This results in `undefined 21420 Malmö` if street address is not exposed, and causes the map to render in the middle of the sea somewhere.

File: src/components/panes/TargetPane.scss [hunk 2]
```diff
@@ -63,6 +77,28 @@
         }
     }
 
+    .TargetPane-personField {
+        &:before {
+            @include icon($fa-var-list);
```
Comment (richardolsson) [inline src/components/panes/TargetPane.scss:82]: I'm not a big fan of this icon repeated over and over again. Maybe something like `$fa-var-info-circle` instead?

**Review by richardolsson — CHANGES_REQUESTED** [review]
I think this looks good when it's working, but I did find a bug (see below). While you're fixing that bug, I also wanted to suggest changing the icon. What do you think?

File: src/components/panes/TargetPane.jsx [hunk 3]
```diff
@@ -21,23 +22,43 @@ export default class TargetPane extends PaneBase {
...
+                ).toArray();
+
+            info = info.concat(customFields);
         }
 
         let map = null;
         if (target.get('zip_code') && target.get('city')) {
+            let location = target.get('zip_code') + '+' + target.get('city')
+            if(target.has('street_address')) {
+                location = target.get('street_address') + '+' + location
```
Comment (richardolsson) [inline src/components/panes/TargetPane.jsx:58]: Semicolon missing

File: src/components/panes/TargetPane.jsx [hunk 4]
```diff
@@ -21,23 +22,43 @@ export default class TargetPane extends PaneBase {
...
+                    </li>
+                ).toArray();
+
+            info = info.concat(customFields);
         }
 
         let map = null;
         if (target.get('zip_code') && target.get('city')) {
+            let location = target.get('zip_code') + '+' + target.get('city')
+            if(target.has('street_address')) {
```
Comment (richardolsson) [inline src/components/panes/TargetPane.jsx:57]: If `street_address` is null, this condition is still true, and the location string ends up being `null+21420+Malmö`.

**Review by richardolsson — CHANGES_REQUESTED** [review]
A few more comments.

First, I'm afraid the bug is still there, but now slightly different.

Second, I think the changes you made in your last commit should have been two separate commits. Making unrelated changes (the icon, the bug fix) in the same commit confuses users of `git blame` and `git log` when trying to understand the history of how some piece of code has evolved.

I really like the look of the new icons though (screenshot below with my address and some dummy custom fields)! 😊 

image

File: src/components/panes/TargetPane.jsx (hunk 4 above)
Comment (niklasva82) [reply src/components/panes/TargetPane.jsx]: ah yeah sorry, replaced with a get. Should not produce that problem now.

**Review by richardolsson — APPROVED** [review]
Confirmed to be working! 💯

**Pull request merged**